tests/tests/test_mtls.py

```diff
@@ -162,19 +141,6 @@ def common_test_mtls_enterprise(self, env, algorithm=None, use_hsm=False):
         )
         env.device.run("update-ca-certificates")
 
-        env.device.put(
-            "server.crt",
-            local_path=os.path.join(certs, "server"),
-            remote_path="/etc/mender",
-        )
-        env.device.put(
-            "cert.crt",
-            local_path=os.path.join(basedir, "cert"),
-            remote_path="/etc/mender",
-        )
-        env.device.run(
-            "cat /etc/mender/server.crt /etc/mender/cert.crt > /tmp/server.crt && mv /tmp/server.crt /etc/mender/server.crt && rm /etc/mender/cert.crt"
-        )
```

Comment:
why is that not needed anymore?

Reply:
This is exactly why I wanted your eyes on it :)

Why was it needed in the first place?

Reply:
Alright, did a little bit of digging. The storage-proxy cert is already a part of the demo-cert now.